components/brave_rewards/browser/rewards_service.cc

```diff
@@ -41,6 +42,15 @@ void RewardsService::RemoveObserver(RewardsServiceObserver* observer) {
 // static
 void RewardsService::RegisterProfilePrefs(PrefRegistrySimple* registry) {
   registry->RegisterStringPref(kRewardsNotifications, "");
+  registry->RegisterTimeDeltaPref(kRewardsNotificationTimerInterval,
```

Comment:
I think this one is not used anymore

Reply:
Yep, I added it and forgot to use it. Fixed!